src/pluginLoader.js

```diff
@@ -0,0 +1,660 @@
+/*
+ * This is a pared down version of hawtio-core in order to remove the outdated
+ * dependency as part the migration from bower to yarn.
+ *
+ * TODO Figure out a better strategy for API discovery so that this can be
+ * removed altogether.
+ *
+ */
+
+// log initialization
+/* globals Logger window console document localStorage $ angular jQuery navigator Jolokia */
+(function() {
+  'use strict';
+
+  Logger.setLevel(Logger.INFO);
+  Logger.storagePrefix = 'hawtio';
```

Comment:
Hm, I don't see `storagePrefix` documented in js-logger or used anywhere below. Can we just remove?

Reply:
I'll try removing and make sure nothing breaks.